cirq-google/cirq_google/engine/simulated_local_job.py

```diff
@@ -72,22 +72,22 @@ def __init__(
                 # close the thread pool once the future is complete.
                 self._thread.shutdown(wait=False)
 
-    def execution_status(self) -> quantum.enums.ExecutionStatus.State:
+    def execution_status(self) -> quantum.ExecutionStatus.State:
         """Return the execution status of the job."""
-        return self._state
+        return self._state  # type: ignore[return-value]
```

Comment:
Can you add a comment for why we need to override the type check? It looks like this should be valid...